torch/nn/_intrinsic/qat/modules/linear_relu.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from torch.nn.qat import Linear as QATLinear
+from torch.nn._intrinsic import LinearReLU2d as NNLinearReLU2d
+from torch.quantization.QConfig import default_qat_qconfig
+import torch.nn.functional as F
+
+class LinearReLU(QATLinear):
+    r"""
+    A LinearReLU module fused from Linear and ReLU modules, attached with
+    FakeQuantize modules for output activation and weight, used in
+    quantization aware training.
+
+    We adopt the same interface as `torch.nn.Linear`, please see https://pytorch.org/docs/stable/nn.html#torch.nn.Linear
+    for documentation.
+
+    Similar to `torch.nn._intrinsic.LinearReLU`, with FakeQuantize modules initialized to
+    default.
+
+    Attributes:
+        observer: fake quant module for output activation, it's called observer
+            to align with post training flow, TODO: rename?
+        weight: fake quant module for weight
+
+    Examples::
+
+        >>> m = nn.qat.LinearReLU(20, 30)
+        >>> input = torch.randn(128, 20)
+        >>> output = m(input)
+        >>> print(output.size())
+        torch.Size([128, 30])
+    """
+    __constants__ = ['bias', 'in_features', 'out_features']
```

Comment:
this is needed only for TorchScript, right? probably delete as it's not a weak module